tests/integration/backend_test.py

```diff
@@ -639,16 +643,16 @@ def test_batching(
 
 @pytest.mark.skipif(skip_remote_tests, reason=REASON)
 @pytest.mark.parametrize(
-    "authenticated_quum_backend",
+    "authenticated_quum_backend_qa",
     [{"device_name": name} for name in pytest.ALL_SYNTAX_CHECKER_NAMES],  # type: ignore
     indirect=True,
 )
 @pytest.mark.parametrize("language", [Language.QASM, Language.QIR])
 @pytest.mark.timeout(120)
 def test_submission_with_group(
-    authenticated_quum_backend: QuantinuumBackend, language: Language
+    authenticated_quum_backend_qa: QuantinuumBackend, language: Language
 ) -> None:
-    b = authenticated_quum_backend
+    b = authenticated_quum_backend_qa
```

Comment:
At line 665 if we changed the default to "DEFAULT" then we wouldn't need to set the environment variable in `build_and_test.yml`.

Reply:
Done in [4bd61b2](https://github.com/CQCL/pytket-quantinuum/pull/357/commits/4bd61b2895cc9d3ff4cd0a82f6601247ad6bc822)